template/types/react-native-web.d.ts

```diff
@@ -0,0 +1,214 @@
+declare module 'react-native' {
+  // The following list is sourced from:
+  // - https://github.com/necolas/react-native-web/blob/0.17.5/packages/react-native-web/src/types/styles.js#L76
```

Comment:
we're using react-native-web 0.19, this is from 0.17 and there are quite a lot of changes and some breaking changes. It would be good to check if those types are correct

Reply:
I updated to the latest version